configs/src/deployments/json/modeMainnet.json

```diff
@@ -0,0 +1,351 @@
+      "blockNumber": 13627066,
+      "deploymentTx": "0x3a633c367c2f4ed2911570377d37a5631bb7e6fad058e0f55140d3210f108c24"
+    },
+    "AdminRepoImplementation": {
+      "address": "0x78744A9664D6A4dDbcd7B73c120535667E739ee3",
+      "blockNumber": 13627066,
+      "deploymentTx": "0x3a633c367c2f4ed2911570377d37a5631bb7e6fad058e0f55140d3210f108c24"
+    },
+    "ManagementDAOProxy": {
+      "address": "0xF3AaA3372EbBf01b923a4Cc98Cd847126b3D73cA",
+      "blockNumber": 13626882,
+      "deploymentTx": "0xa25407d3c0fd9db57545a65f70f25eefd60ee1fcc161038539d67662f8dcf53f"
+    },
+    "ManagementDAOImplementation": {
+      "address": "0x741061B01d3A95a7734B594f1884b6c30E0fDA96",
+      "blockNumber": 13626879,
+      "deploymentTx": "0x794ba6998e7ac7cee6c7fa505edceaf975c4191541649cf364fd921b55eaa3de"
+    },
+    "MultisigRepoProxy": {
+      "address": "0x7553E6Fb020c5740768cF289e603770AA09b7aE2",
+      "blockNumber": 13627076,
+      "deploymentTx": "0x446005fbd528deaf5b2ad35b146f13ffa7cacaa96abbe9292b6fa19451a939f7"
+    },
+    "MultisigRepoImplementation": {
+      "address": "0x78744A9664D6A4dDbcd7B73c120535667E739ee3",
+      "blockNumber": 13627031,
+      "deploymentTx": "0x7058071c479c267c687f6f3a80710003c28f867abe2ec819b813bc107e8ee492"
+    },
+    "TokenVotingRepoProxy": {
+      "address": "0x1AeD2BEb470aeFD65B43f905Bd5371b1E4749d18",
+      "blockNumber": 13627053,
+      "deploymentTx": "0xca53ea495e6b58ee198c9ebc26cad6ab61d6f120ca90cdad91feba4d3c0fe5ca"
+    },
+    "TokenVotingRepoImplementation": {
+      "address": "0x78744A9664D6A4dDbcd7B73c120535667E739ee3",
+      "blockNumber": 13627023,
+      "deploymentTx": "0x6e961c3c4558e8bab2f74e3ffe60c56afed1a6ef4fa2a85db3ed6cc3d8344fc8"
+    }
+  },
+  "v1.4.0": {
+    "AdminSetup": {
+      "address": "0x264308C03feAfA071C97b73b09E911530CCCd216",
+      "blockNumber": 21115240,
+      "deploymentTx": "0xb95310e3a9ed1eb74f9eff51f9ebf2ebf031d9c02d5bd1007b912ebadf3a52d5"
+    },
+    "AdminSetupImplementation": {
+      "address": "0x1CA2E42e3eECf1b39BE23a424Df0585e887275a8",
+      "blockNumber": 21115240,
+      "deploymentTx": "0xb95310e3a9ed1eb74f9eff51f9ebf2ebf031d9c02d5bd1007b912ebadf3a52d5"
+    },
+    "DAOBase": {
+      "address": "0x4C8288220e7Bf31e6dEe16fB58d652E024529682",
+      "blockNumber": 21117123,
+      "deploymentTx": "0x20f712a67e542b60597796d4340796a292176291aa272a66b8fb6425feac38b1"
+    },
+    "DAOFactory": {
+      "address": "0xBB1F7ACf2068Ec3F669FDCB4d0Ad9a70E3E418d6",
+      "blockNumber": 21117123,
+      "deploymentTx": "0x20f712a67e542b60597796d4340796a292176291aa272a66b8fb6425feac38b1"
+    },
+    "DAORegistryProxy": {
+      "address": "0xB5146Fd572C669ABC353902e43F47fda4609E38A",
+      "blockNumber": 13626963,
+      "deploymentTx": "0x5e6936d94e5fda3569d25c66337bd85a0086e0d1b7dcff198b76139b7ebe3c59"
+    },
+    "DAORegistryImplementation": {
+      "address": "0x76d72457990e07dd889Bd8F19a02feB89d11A946",
```

Comment:
`DAORegistryProxy` is currently pointing at `0xf2d594F3C93C19D7B1a6F15B5489FFcE4B01f7dA`

Reply:
it will be pointing at what ever 1.4 upgrade will set them to
please check log and compare against the log
this set of address are under item 1.4.0